### PR TITLE
Add ml alerts examples

### PR DESCRIPTION
**Pull request by andrewm4894**
##### Summary

This PR adds some examples of how to use the ML to define some ML based alerts should users wish to do so.

Currently [Anomaly Advisor](https://learn.netdata.cloud/docs/cloud/insights/anomaly-advisor) is passive and does not actively alert (by design). This PR would document how users who would like to trigger alerts based on anomaly rate can do so within the existing health configurations of Netdata.

##### Test Plan

I have been using sets of alerts like this in our devml and ml demo spaces, for example: 

https://app.netdata.cloud/spaces/ml-demo-space/rooms/general/overview#chartName=menu_alarms_submenu_value&correlation=false&after=1655457667000&before=1655464867000&utc=Europe%2FLondon&offset=%2B1&timezoneName=Edinburgh%2C%20London&metrics_correlation=false&appliedFilteredIds=&reachableNodesIds=&staleNodesIds=&unreachableNodesIds=&filteredNodesSearchParams=

When we have anomalies on the system:

image

You can see that the ML alerts follow soon after: 

image

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Review by vkalintiris — DISMISSED** [review]

File: health/REFERENCE.md [hunk 1]
```diff
@@ -895,6 +895,48 @@ lookup: mean -10s of user
 
 Since [`z = (x - mean) / stddev`](https://en.wikipedia.org/wiki/Standard_score) we create two input alarms, one for `mean` and one for `stddev` and then use them both as inputs in our final `cpu_user_zscore` alarm.
 
+### Example 8 - Anomaly rate based CPU dimensions alarm
+
+Warning if 5 minute rolling anomaly rate for any CPU dimension is above 5%, critical if it goes above 20%:
```
Comment (ilyam8) [inline health/REFERENCE.md:900]: Can we make anomaly rate a hyperlink that redirects to a doc that describes what "anomaly rate" is? As it is now it implies a person has knowledge of the subject.

Comment (andrewm4894) [reply health/REFERENCE.md]: done

**Review by vkalintiris — APPROVED** [review]

File: health/health.d/ml.conf [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#       on: system.cpu
+#       os: linux
+#    hosts: *
+#   lookup: average -5m anomaly-bit of *
+#     calc: $this
+#    units: %
+#    every: 30s
+#     warn: $this > (($status >= $WARNING)  ? (5) : (20))
+#     crit: $this > (($status == $CRITICAL) ? (20) : (100))
+#     info: rolling 5min anomaly rate for system.cpu chart
```
Comment (thiagoftsm) [inline health/health.d/ml.conf:36]: I used the wrong button, so I am writing again, please, add a blank line to remove this github warning.

Comment (ilyam8) [reply health/health.d/ml.conf:36]: @thiagoftsm I think we can live with that 😄

**Pull request merged**